pkg/client/info_test.go

```diff
@@ -102,12 +103,12 @@ func TestMultiClientInfo(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	mFactory := mocks.NewMockProjectClientFactory(ctrl)
 	mFactory.EXPECT().List(gomock.Any()).Return([]client.Client{&defaultClient1, &defaultClient2}, nil)
-	mFactory.EXPECT().DefaultProject().Return("test1")
+	//mFactory.EXPECT().DefaultProject().Return("test1")
```

Comment:
Any reason to comment out instead of removing?

Reply:
oh no! was testing the fix

Reply:
will delete